### DISCUSSION TITLE
How can I access the the Information manually in the script part, if a FormKit Group is valid or not?

### DISCUSSION THREAD

**Discussion by silviaAut** [formkit/formkit#739] [General]
Hello,
I have this code:
![image](https://github.com/formkit/formkit/assets/6195977/d815ca74-6ce1-4894-8785-106614c2518b)

And I want to trigger a validation manually from the script, but how can I do that?

Best regards,
Silvia

**Comment by fenilli** [comment]
You can get the node inside the script using the `getNode` method, there you have access to all node properties.

Also the thing is, you don't need to trigger validation manually as it is always being computed, if any children is in a invalid state, the group will be invalid at that point.
